Library/Homebrew/test/support/helper/minitest/assertions.rb

```diff
@@ -0,0 +1,19 @@
+module MiniTest
+  module Assertions
```

Comment:
What's the advantage of monkey patching this module compared to the previous approach?